### PR TITLE
doc: Add Release Notes and Known Issues for CS2

### PR DESCRIPTION
**Pull request by FrancescoSer**
Added release notes for 2.6.99-cs2
Added known issues for 2.6.99-cs2

### CODE REVIEW

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following ESB samples:
+
+  * :ref:`esb_prx` sample
+  * :ref:`esb_ptx` sample
+
+* Added fast switching between radio states for the nRF54H20 SoC.
+* Added fast radio channel switching for the nRF54H20 SoC.
+
+Multicore samples
+-----------------
```
Comment (gmarull) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:86]: empty block

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  * CLOCK
+  * GPIO
+  * GRTC
+  * I2C
+  * MRAM
+  * PWM
+  * RADIO (Bluetooth Low Energy and 802.15.4)
+  * SPI
+  * UART
+  * USB
```
Comment (gmarull) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:41]: is USB supported?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+  * :ref:`radio_test` sample
+
+* Updated:
+
+  * The CLI command ``fem tx_power_control <tx_power_control>`` replaces ``fem tx_gain <tx_gain>`` .
+    This change applies to the sample built with the :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
+
+Other samples
+-------------
+
```
Comment (gmarull) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:102]: empty

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+Current nRF54H20 DK limitations
+===============================
+
+
+
+General
+-------
+
+
```
Comment (gmarull) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:153]: empty

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 3 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:102]: This is still a draft

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 4 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:153]: This is still a draft

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+   git log --oneline 0051731a41 ^23cf38934c
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^0051731a41
+
+The current |NCS| main branch is based on revision ``0051731a41`` of Zephyr.
+
```
Comment (FrancescoSer) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:132]: @gmarull @carlescufi @rlubos Could you please check the commit hashes here?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 6]
```diff
@@ -0,0 +1,167 @@
+.. _ncs_release_notes_2699_cs1:
+
+|NCS| v2.6.99-cs2 Release Notes
```
Comment (FrancescoSer) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:3]: @KAGA164 @dchat-nordic Could you please review this PR?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 2 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:41]: @carlescufi @kapi-no Any insight here?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 1 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:89]: fixed

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+Enhanced ShockBurst samples
+---------------------------
+
+* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following ESB samples:
+
+  * :ref:`esb_prx` sample
+  * :ref:`esb_ptx` sample
+
+* Added fast switching between radio states for the nRF54H20 SoC.
+* Added fast radio channel switching for the nRF54H20 SoC.
```
Comment (grochu) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:86]: These lines may fit better the protocol section, but can stay here too.

**Review by grochu — COMMENTED** [review]
I added some comments for clarification. In general are NFC and BLE support relevant for this devtag or shall we skip them in the docs?

**Comment by kato-nordic** [comment]
NFC and BLE not relevant – not needed for RadioTest example?


KARL HELMER TORVMARK | Technical Product Manager
M +47 48 21 25 67 | Oslo, Norway
nordicsemi.com<http://www.nordicsemi.com/> | devzone.nordicsemi.com<https://devzone.nordicsemi.com/>
Facebook<https://www.facebook.com/nordicsemiconductor/> | LinkedIn<https://www.linkedin.com/company/nordic-semiconductor-asa/> | Twitter<https://twitter.com/NordicTweets> | YouTube<https://www.youtube.com/user/NordicSemi> | Instagram<https://www.instagram.com/nordicsemi/>

[Nordic_logo_signature]<http://www.nordicsemi.com/>


[Image]<https://www.nordicsemi.com/nrf54H20>

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+  * GRTC
+  * I2C
+  * MRAM
+  * PWM
+  * RADIO (Bluetooth Low Energy and 802.15.4)
+  * SPI
+  * UART
+  * USB
+  * WDT
+  * NFCT
```
Comment (grochu) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:43]: Shouldn't it be in the alphabetical order?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+
+  * :ref:`direct_test_mode` sample
+
+    * Added experimental support for the HCI interface.
+    * Changed internal sample API.
+
+Enhanced ShockBurst samples
+---------------------------
+
+* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following ESB samples:
```
Comment (grochu) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:83]:
```suggestion
* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following ESB samples:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following Bluetooth samples:
```
Comment (grochu) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:73]:
```suggestion
* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following Bluetooth samples:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following peripheral samples:
+
+  * :ref:`multicore_hello_world` sample
+
+Peripheral samples
+------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following peripheral samples:
```
Comment (grochu) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:98]:
```suggestion
* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following peripheral samples:
```

**Review by grochu — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+
+Zephyr samples
+--------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` in the following Zephyr samples:
+
+  * :zephyr:code-sample:`blinky`
+  * :ref:`zephyr:hello_world`
+  * :zephyr:code-sample:`fade-led`
+
```
Comment (maje-emb) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:142]: @grochu @KAGA164 Shouldn't we also add a bluetooth observer?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 10 above)
Comment (maje-emb) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:73]: The following samples support the nrf54h20dk_nrf54h20_cpuapp target:
- central_and_peripheral_hr
- central_bas
- central_hids
- central_hr_coded
- central_uart
- multiple_adv_sets
- peripheral_ams_client
- peripheral_ancs_client
- peripheral_bms
- peripheral_cgm
- peripheral_cts_client
- peripheral_gatt_dm
- peripheral_hids_keyboard
- peripheral_hids_mouse
- peripheral_hr_coded
- peripheral_lbs
- peripheral_nfc_pairing
- peripheral_rscs
- peripheral_status
- peripheral_uart
- shell_bt_nus
- throughput

Comment (maje-emb) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:73]: Shouldn't we list supported samples by nRF54L15?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` board.
+* Added fast switching between radio states for the nRF54H20 SoC.
+* Added fast radio channel switching for the nRF54H20 SoC.
+
+Samples
+=======
+
```
Comment (maje-emb) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:67]: The following NFC samples support nrf54h20dk_nrf54h20_cpuapp target:
- record_launch_app
- record_text
- shell
- tnep_tag
- writable_ndef_msg

and the following support nrf54l15:
- record_launch_app
- record_text
- shell
- system_off
- tnep_tag
- writable_ndef_msg

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 12 above)
Comment (grochu) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:142]: I think it's not in scope for NCS 2.6.99-cs2

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 10 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:73]: I don't think these are in scope of this release

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 13 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:67]: Moonlight and these samples are not in scope of this release, I believe

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 14]
```diff
@@ -171,7 +171,7 @@ Protocol support
 The following table indicates the software maturity levels of the support for each :ref:`protocol <protocols>`:
 
 .. sml-table:: top_level
-   :insert-values: [("Bluetooth","nRF54L15","Experimental"), ("Bluetooth Mesh","nRF54L15","-"), ("Matter","nRF54L15","Experimental"), ("Sidewalk","nRF54L15","Experimental"), ("Thread","nRF54L15","Experimental")]
+   :insert-values: [("Bluetooth","nRF54L15","Experimental"), ("Bluetooth Mesh","nRF54L15","-"), ("Matter","nRF54L15","Experimental"), ("Sidewalk","nRF54L15","Experimental"), ("Thread","nRF54L15","Experimental"), ("Bluetooth","nRF54H20","Experimental"), ("Wi-Fi","nRF54H20","-")]
```
Comment (FrancescoSer) [inline doc/nrf/releases_and_maturity/software_maturity.rst:174]: @ppryga-nordic

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 15]
```diff
@@ -190,6 +190,7 @@ The following table indicates the software maturity levels of the support for ea
 .. toggle::
 
   .. sml-table:: bluetooth
+     :insert-values: [("Bluetooth LE Peripheral/Central","nRF54H20","Experimental"), ("Connectionless/Connected CTE Transmitter","nRF54H20","Experimental"), ("LE Coded PHY","nRF54H20","Experimental")]
```
Comment (ppryga-nordic) [inline doc/nrf/releases_and_maturity/software_maturity.rst:193]: @FrancescoSer  shouldn't it be removed? I think if changes nRF54H20 column content that LLPM is stated as not available.

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+  * :ref:`direct_test_mode` sample
+
+    * Added experimental support for the HCI interface.
+    * Changed internal sample API.
+
+Enhanced ShockBurst samples
+---------------------------
+
+* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following ESB samples:
```
Comment (grochu) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:86]: Shall we align the link with as in other samples' sections?
```suggestion
* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following ESB samples:
```
```suggestion
* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following ESB samples:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` board.
+* Added fast switching between radio states for the nRF54H20 SoC.
+* Added fast radio channel switching for the nRF54H20 SoC.
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:66]:
```suggestion
* Added:

   * Support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` board.
   * Fast switching between radio states for the nRF54H20 SoC.
   * Fast radio channel switching for the nRF54H20 SoC.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 18]
```diff
@@ -0,0 +1,157 @@
...
+
+Bluetooth samples
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following Bluetooth sample:
+
+  * :ref:`direct_test_mode` sample
+
+    * Added experimental support for the HCI interface.
+    * Changed internal sample API.
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:81]:
```suggestion
    * Updated the internal sample API.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 16 above)
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:86]:
```suggestion
* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build target ``nrf54h20dk_nrf54h20_cpurad`` in the following ESB samples:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 19]
```diff
@@ -0,0 +1,157 @@
...
+
+* Added support for the :ref:`nRF54H20 DK <zephyr:nrf54h20dk_nrf54h20>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following ESB samples:
+
+  * :ref:`esb_prx` sample
+  * :ref:`esb_ptx` sample
+
+Multicore samples
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpuapp`` in the following multicore sample:
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:94]:
```suggestion
* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build target ``nrf54h20dk_nrf54h20_cpuapp`` in the following multicore sample:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 20]
```diff
@@ -0,0 +1,157 @@
...
+
+  * :ref:`multicore_hello_world` sample
+
+Peripheral samples
+------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following peripheral sample:
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:101]:
```suggestion
* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build target ``nrf54h20dk_nrf54h20_cpurad`` in the following peripheral sample:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 21]
```diff
@@ -0,0 +1,157 @@
...
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following Bluetooth sample:
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:76]:
```suggestion
* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build target ``nrf54h20dk_nrf54h20_cpurad`` in the following Bluetooth sample:
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 22]
```diff
@@ -0,0 +1,157 @@
...
+------------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` with build targets ``nrf54h20dk_nrf54h20_cpurad`` in the following peripheral sample:
+
+  * :ref:`radio_test` sample
+
+* Updated:
+
+  * The CLI command ``fem tx_power_control <tx_power_control>`` replaces ``fem tx_gain <tx_gain>`` .
+    This change applies to the sample built with the :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:108]:
```suggestion
    This change applies to the sample built with the Kconfig option :ref:`CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC <CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC>` set to ``n``.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 2 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:42]: @gmarull @carlescufi @kapi-no @pdunaj @miha-nordic Any insight here?

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 5 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:133]: @gmarull @carlescufi @rlubos Please review the hashes here

File: doc/nrf/releases_and_maturity/software_maturity.rst (hunk 15 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/software_maturity.rst:193]: No, LLPM  is stated as not available even without this line. This line just changes what was there to experimental. Default is supported. What's the correct info there? Is it available?

**Review by ppryga-nordic — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 2 above)
Comment (ns-tolu) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:47]: not supported yet

**Review by ns-tolu — CHANGES_REQUESTED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 23]
```diff
@@ -0,0 +1,170 @@
...
+
+  * The ``ug_nrf54h`` section.
+  * The ``Migration notes for |NCS| v2.6.99_cs2 for v2.4.99-cs3 users`` section.
+
+* Updated the table listing the :ref:`boards included in sdk-zephyr <app_boards_names_zephyr>` with the nRF54H20 DK board.
+
+Current nRF54H20 DK Limitations
+*******************************
+
+* On the nRF54H20 DK Revision PCA10175, the **ON** and **OFF** markings for the power switch on the PCB are switched.
```
[the remark this answers is not in the document]
Comment (miha-nordic) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:169]: It is also applicable for 0.7.2 but not sure if customers will get that version.

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 24]
```diff
@@ -0,0 +1,170 @@
...
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Zephyr samples
+--------------
+
+* Added support for the :ref:`nRF54H20 DK <ug_nrf54h20_gs>` in the following Zephyr samples:
+
+  * :zephyr:code-sample:`blinky`
```
Comment (miha-nordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:148]: Should that be blinky_pwm? @ns-tolu

**Review by miha-nordic — APPROVED** [review]

File: doc/nrf/releases_and_maturity/release_notes.rst [hunk 25]
```diff
@@ -20,7 +20,7 @@ This page is included only in the latest documentation, because it might contain
    :maxdepth: 1
    :caption: Subpages:
 
-   releases/release-notes-changelog
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/release_notes.rst:23]: Remove bullet point in line 15 above.

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 23 above)
[the remark this answers is not in the document]
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:169]: Ok, I'll mention v0.7.x

**Review by grochu — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 26]
```diff
@@ -0,0 +1,170 @@
...
+  * UART
+  * USB
+  * WDT
+
+Working with nRF54H Series
+==========================
+
+* Added:
+
+  * The :ref:`ug_nrf54h20_gs` page.
```
[the remark this answers is not in the document]
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:55]: There are 2 items under "added"

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 27]
```diff
@@ -0,0 +1,170 @@
...
+=============
+
+..
+  ## Update documentation links after cherry-pick
+  ## Update software maturity table
+
+* Added:
+
+  * The ``ug_nrf54h`` section.
+  * The ``Migration notes for |NCS| v2.6.99_cs2 for v2.4.99-cs3 users`` section.
```
[the remark this answers is not in the document]
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:162]: I'm not sure about this one, there are actually 3 pages: one container page and 2 user guides

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 28]
```diff
@@ -0,0 +1,159 @@
...
+Documentation
+=============
+
+..
+  ## Update documentation links after cherry-pick
+  ## Update software maturity table
+
+* Added:
+
+  * The ``ug_nrf54h`` section.
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:159]: Link missing for these two entries.

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 29]
```diff
@@ -0,0 +1,168 @@
...
+
+Working with nRF54H Series
+==========================
+
+* Added:
+
+  * The :ref:`ug_nrf54h20_gs` page.
+
+..
+  ##ADD the rest of the doc pages
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:57]: More entries to be added here?

File: doc/nrf/conf.py [hunk 30]
```diff
@@ -28,7 +28,7 @@
 project = "nRF Connect SDK"
 copyright = "2019-2024, Nordic Semiconductor"
 author = "Nordic Semiconductor"
-version = release = "2.6.99"
+version = release = "2.6.99-cs2"
```
Comment (umapraseeda) [inline doc/nrf/conf.py:31]: Same update missing for nrfxlib/conf.py

**Review by b-gent — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst (hunk 29 above)
Comment (FrancescoSer) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:57]: Yes, when the other PRs will be merged

**Review by ns-tolu — APPROVED** [review]

File: doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst [hunk 31]
```diff
@@ -0,0 +1,168 @@
...
+
+Zephyr
+======
+
+..
+  ##update commits
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``0051731a41fa2c9057f360dc9b819e47b2484be5``, with some |NCS| specific additions.
```
Comment (shanthanordic) [inline doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:123]: image

Comment (shanthanordic) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:123]: @rlubos

Comment (rlubos) [reply doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs2.rst:115]: This looks correct, it corresponds to the merge base used in the last upmerge on this branch.

**Review by rlubos — APPROVED** [review]
Zephyr SHAs look correct.

**Pull request merged**